tools/pybind11Tools.cmake

```diff
@@ -1,6 +1,6 @@
 # tools/pybind11Tools.cmake -- Build system for the pybind11 modules
 #
-# Copyright (c) 2015 Wenzel Jakob <wenzel@inf.ethz.ch>
+# Copyright (c) 2020 Wenzel Jakob <wenzel.jakob@epfl.ch>
```

Comment:
Far from an expert, but I thought I most often see `2015-2020` in copyright messages?

E.g. https://softwareengineering.stackexchange.com/questions/227812/why-do-we-need-years-in-a-software-license or https://stackoverflow.com/questions/2390230/do-copyright-dates-need-to-be-updated

Reply:
Shouldn't we update those everywhere?

Reply:
I think so, yes. But I just saw the change here.

Reply:
Not really a big deal, the first stackoverflow link you posted says that it's kind of unnecessary. Just made an edit because I spotted an ancient email address that no longer works.